### PR TITLE
Blog: 2024 April updates

### PR DESCRIPTION
**Pull request by psiinon**
<details><summary>Screenshot</summary>
<p>

Screenshot 2024-05-01 at 10-22-56 ZAP Updates - April 2024

</p>
</details>

### CODE REVIEW

**Review by ricekot — APPROVED** [review]
LGTM. Thanks!

File: site/content/blog/2024-05-01-zap-updates-april-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+There is a brand new ZAP GitHub action - the 
+[ZAP Automation Framework Scan](https://github.com/marketplace/actions/zap-automation-framework-scan).
+This allows you run an [Automation Framework](/docs/automate/automation-framework/) plan in a GitHub action.
+
+Why is this important?
+Because the Automation Framework provides a [great balance](/docs/getting-further/automation/automation-options/)
+between ease of use and flexibility + functionality.
+ 
+If you want to perform any non trivial automation with ZAP then the
+[Automation Framework](/docs/automate/automation-framework/) is probably your best best.
```
Comment (kingthorin) [inline site/content/blog/2024-05-01-zap-updates-april-2024/index.md:68]: bet

File: site/content/blog/2024-05-01-zap-updates-april-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+## Ongoing Work
+
+### Funding
+
+Funding continues to be a significant focus, and I am delighted to announce that the first ZAP 
+[Support Package Contracts](/support/#support-packages) have been signed!
+
+If you need ZAP support then head over to the [Support](/support/) page.
+All of the money raised for via the professional services will help fund ZAP development - ZAP is a non-profit open source project.
```
Comment (kingthorin) [inline site/content/blog/2024-05-01-zap-updates-april-2024/index.md:81]: Drop 'for' in front of 'via'

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**